src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -329,25 +329,25 @@
   <data name="Nav_Rendering.Content" xml:space="preserve">
     <value>Rendering</value>
   </data>
-  <data name="Profile_AcrylicOpacity.Text" xml:space="preserve">
+  <data name="Profile_AcrylicOpacity.HeaderText" xml:space="preserve">
     <value>Acrylic opacity</value>
   </data>
-  <data name="Profile_AcrylicOpacity.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
+  <data name="Profile_AcrylicOpacity.HelpText" xml:space="preserve">
     <value>Sets the transparency of the window.</value>
   </data>
   <data name="Profile_Advanced.Header" xml:space="preserve">
     <value>Advanced</value>
   </data>
-  <data name="Profile_AltGrAliasing.Header" xml:space="preserve">
+  <data name="Profile_AltGrAliasing.HeaderText" xml:space="preserve">

```

Comment:
It seems like you could save a HUGE loc burden by just making the dependencyproperty `Header`, right? Just... same key, same resource, bind it into the template.

Reply:
Yeah. So this is dumb and annoying. The "Header" dependency property is apparently already in use (see below).
```
1>D:\Terminal\src\cascadia\TerminalSettingsEditor\Generated Files\XamlTypeInfo.g.cpp(493,44): error C2995: 'winrt::Windows::Foundation::IInspectable winrt::Microsoft::Terminal::Settings::Editor::implementation::GetReferenceTypeMember_Header(const winrt::Windows::Foundation::IInspectable &)': function template has already been defined
1>D:\Terminal\src\cascadia\TerminalSettingsEditor\Generated Files\XamlTypeInfo.g.cpp(463): message : see declaration of 'winrt::Microsoft::Terminal::Settings::Editor::implementation::GetReferenceTypeMember_Header'
1>D:\Terminal\src\cascadia\TerminalSettingsEditor\Generated Files\XamlTypeInfo.g.cpp(1387,6): error C2995: 'void winrt::Microsoft::Terminal::Settings::Editor::implementation::SetReferenceTypeMember_Header(const winrt::Windows::Foundation::IInspectable &,const winrt::Windows::Foundation::IInspectable &)': function template has already been defined
```
_But_ if we try and use "Header", we fail catastrophically:
- adding `TabTitle.Header` to the resw --> `'Unable to resolve property 'Header' while processing properties for Uid 'Profile_TabTitle'. [Line: 138 Position: 69]'`
- `Header()` does not exist in SettingContainer.cpp

Reply:
Aha! Issue was that `Header` is an `IInspectable`. Fixed!